<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskUserNameAndPassword</class>
 <widget class="QWidget" name="AskUserNameAndPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User name and password</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textMessage">
     <property name="text">
      <string>Please enter your user name and password from Evernote service</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userNameEnterBuddy">
     <property name="text">
      <string>User &amp;name:</string>
     </property>
     <property name="buddy">
      <cstring>userNameEnterForm</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="userNameEnterForm"/>
   </item>
   <item>
    <widget class="QLabel" name="userPasswordEnterBuddy">
     <property name="text">
      <string>User &amp;password:</string>
     </property>
     <property name="buddy">
      <cstring>userPasswordEnterForm</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="userPasswordEnterForm"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
